.github/workflows/build-plugin-archive.yml

```
@@ -98,6 +103,20 @@ jobs:
               echo "archive-name=${{ github.event.repository.name }}" >> $GITHUB_OUTPUT
           fi
 
+      - name: Set up plugin dir name
```

```suggestion
      - name: Set up plugin folder name
```

While on it, please also fix the step name in line 95 to something like "Set up archive name". Thanks!